package.json

```diff
@@ -83,7 +83,7 @@
     "d3-hexbin": "^0.2.2",
     "d3-request": "^1.0.6",
     "d3-scale": "^1.0.6",
-    "deck.gl": "^7.1.5",
+    "deck.gl": "7.1.5",
```

Comment:
let's lock luma.gl as well. Since they come in pairs

Reply:
Done; locked to the version that install with this version of deck.gl (7.1.1, as already specified)